docs/extensions/triggers.rst

```diff
@@ -0,0 +1,94 @@
+.. py:currentmodule:: coc
+
+.. _triggers_extension:
+
+Triggers Extension
+=======================
+
+Overview
+--------
+
+coc.py's events are an extremely powerful framework, but they are not particularly well suited for periodic bulk-update style tasks, and employing the use of ``APScheduler`` or similar modules feels excessive for such a simple job. That is where the triggers extension for coc.py comes into play.
```

Comment:
I would prefer all these long lines to be splitted into multiple ones not longer than 120 characters tbh

Reply:
For code, absolutely. For docs, I don't view it quite as strictly, but I'm fine with changing it up